src/WattleScript.Interpreter/Tree/MemberUtilities.cs

```diff
@@ -12,10 +13,11 @@ class MemberUtilities
             "__tostring",
         };
 
-        private static readonly (MemberModifierFlags a, MemberModifierFlags b, string msg)[] flagConflicts =
+        private static readonly (MemberModifierFlags a, MemberModifierFlags? b, WattleMemberType[] appliesTo, string msg)[] flagConflicts =
```

Comment:
Change WattleMemberType to flags, with Any being all values. This gets rid of the array and is a bit cleaner.

Reply:
Fixed in 8462beb